src/components/Result/Result.js

```diff
@@ -1,16 +1,24 @@
-import React, { useState } from "react";
+import React from "react";
 import "./Result.css";
 import { NavLink } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { increment } from "../../features/daySlice";
 
 function Result() {
+  const dayNum = useSelector((state) => state.day.value);
+  const dispatch = useDispatch();
+
+  const getNextDay = dayNum < 7 ? `/${dayNum}/day` : "/finale";
+
   return (
     <>
+      <p>Day {dayNum} results</p>
       <p>You chose free buffet. You floofy loaf!</p>
-      <NavLink to="/">
-        <button>CONTINUE</button>
+      <NavLink to={getNextDay}>
+        <button onClick={() => dispatch(increment())}>CONTINUE</button>
```

Comment:
I was curious how you would trigger changing days, since going off of actual 24 hrs day changes doesn't feel like the good fit if we want people to be able to experience the full glory of this project without having to spend a full week.   I like this implementation of users just being able to click through!